libqtile/backend/x11/xcbq.py

```diff
@@ -541,18 +588,24 @@ def keycode_to_keysym(self, keycode, modifier):
 
     def create_window(self, x, y, width, height):
         wid = self.conn.generate_id()
+
+        value_mask = CW.BackPixmap | CW.BorderPixel | CW.EventMask | CW.Colormap
+        values = [
+            xcffib.xproto.BackPixmap._None,
+            0,
```

Comment:
Why do we need to specify the BorderPixel now?

Reply:
See: https://lists.freedesktop.org/archives/xcb/2017-October/011047.html

Reply:
Interesting. Thanks!

Reply:
I did also check that this was needed and it does stop working if you remove it.